src/app/pages/psbt-request/hooks/use-psbt-decoded-request.tsx

```diff
@@ -40,13 +36,20 @@ export function usePsbtDecodedRequest({
   const [showAdvancedView, setShowAdvancedView] = useState(false);
   const network = useCurrentNetwork();
   const nativeSegwitSigner = useCurrentAccountNativeSegwitIndexZeroSigner();
-  const unsignedUtxos = useOrdinalsAwareUtxoQueries(unsignedInputs).map(query => query.data);
+  // Use the unsigned inputs txid and index to get the utxo data
+  const unsignedUtxos = useGetBitcoinTransactionQueries(unsignedInputs)
+    .map(query => query.data)
+    .map((input, i) => input?.vout[unsignedInputs[i].index]);
+  // Get utxos with possible inscriptions separately
+  const unsignedUtxosWithInscriptions = useOrdinalsAwareUtxoQueries(unsignedInputs).map(
+    query => query.data
+  );
```

Comment:
I wonder if this could be merged somehow so that we have one resulting type, instead of more inputs? 

I think there are two potential issues having two related, utxo inputs:
- Were another asset type to come along, we'd have to add a third input. Lots of the code becomes aware of the implementation detail (that the ordinals aware utxo query doesn't work on testnet). When it could be taken care of in one place in the code, if we create a separate type first.
- There's an unclear relationship that the indexes have to align, which seems prone to error, and unnecessary if we were to first create a type first.